internal_review/demographics/borough/ACS_PUMS_demographics_0812.csv

```diff
@@ -0,0 +1,6 @@
+borough,lep_count,lep_count_moe,lep_count_cv,lep_pct,lep_pct_moe,lep_anh_count,lep_anh_count_moe,lep_anh_count_cv,lep_anh_pct,lep_anh_pct_moe,lep_bnh_count,lep_bnh_count_moe,lep_bnh_count_cv,lep_bnh_pct,lep_bnh_pct_moe,lep_hsp_count,lep_hsp_count_moe,lep_hsp_count_cv,lep_hsp_pct,lep_hsp_pct_moe,lep_wnh_count,lep_wnh_count_moe,lep_wnh_count_cv,lep_wnh_pct,lep_wnh_pct_moe,fb_count,fb_count_moe,fb_count_cv,fb_pct,fb_pct_moe,fb_anh_count,fb_anh_count_moe,fb_anh_count_cv,fb_anh_pct,fb_anh_pct_moe,fb_bnh_count,fb_bnh_count_moe,fb_bnh_count_cv,fb_bnh_pct,fb_bnh_pct_moe,fb_hsp_count,fb_hsp_count_moe,fb_hsp_count_cv,fb_hsp_pct,fb_hsp_pct_moe,fb_wnh_count,fb_wnh_count_moe,fb_wnh_count_cv,fb_wnh_pct,fb_wnh_pct_moe,age_popu16_count,age_popu16_count_moe,age_popu16_count_cv,age_popu16_pct,age_popu16_pct_moe,age_p16t64_count,age_p16t64_count_moe,age_p16t64_count_cv,age_p16t64_pct,age_p16t64_pct_moe,age_p65pl_count,age_p65pl_count_moe,age_p65pl_count_cv,age_p65pl_pct,age_p65pl_pct_moe,age_popu16_anh_count,age_popu16_anh_count_moe,age_popu16_anh_count_cv,age_popu16_anh_pct,age_popu16_anh_pct_moe,age_popu16_bnh_count,age_popu16_bnh_count_moe,age_popu16_bnh_count_cv,age_popu16_bnh_pct,age_popu16_bnh_pct_moe,age_popu16_hsp_count,age_popu16_hsp_count_moe,age_popu16_hsp_count_cv,age_popu16_hsp_pct,age_popu16_hsp_pct_moe,age_popu16_wnh_count,age_popu16_wnh_count_moe,age_popu16_wnh_count_cv,age_popu16_wnh_pct,age_popu16_wnh_pct_moe,age_p16t64_anh_count,age_p16t64_anh_count_moe,age_p16t64_anh_count_cv,age_p16t64_anh_pct,age_p16t64_anh_pct_moe,age_p16t64_bnh_count,age_p16t64_bnh_count_moe,age_p16t64_bnh_count_cv,age_p16t64_bnh_pct,age_p16t64_bnh_pct_moe,age_p16t64_hsp_count,age_p16t64_hsp_count_moe,age_p16t64_hsp_count_cv,age_p16t64_hsp_pct,age_p16t64_hsp_pct_moe,age_p16t64_wnh_count,age_p16t64_wnh_count_moe,age_p16t64_wnh_count_cv,age_p16t64_wnh_pct,age_p16t64_wnh_pct_moe,age_p65pl_anh_count,age_p65pl_anh_count_moe,age_p65pl_anh_count_cv,age_p65pl_anh_pct,age_p65pl_anh_pct_moe,age_p65pl_bnh_count,age_p65pl_bnh_count_moe,age_p65pl_bnh_count_cv,age_p65pl_bnh_pct,age_p65pl_bnh_pct_moe,age_p65pl_hsp_count,age_p65pl_hsp_count_moe,age_p65pl_hsp_count_cv,age_p65pl_hsp_pct,age_p65pl_hsp_pct_moe,age_p65pl_wnh_count,age_p65pl_wnh_count_moe,age_p65pl_wnh_count_cv,age_p65pl_wnh_pct,age_p65pl_wnh_pct_moe,age_median_median,age_median_median_moe,age_median_median_cv,age_median_anh_median,age_median_anh_median_moe,age_median_anh_median_cv,age_median_bnh_median,age_median_bnh_median_moe,age_median_bnh_median_cv,age_median_hsp_median,age_median_hsp_median_moe,age_median_hsp_median_cv,age_median_wnh_median,age_median_wnh_median_moe,age_median_wnh_median_cv
```

Comment:
output column names look good but see first comment, missing population columns in all outputs

Reply:
I mean the data points for the denominators for foreign-born population pct and the age group pct

Reply:
ah I see.  Apologies.  Yes, those need to be included.  Thanks for flagging.